modules/social_features/social_tagging/src/SocialTaggingService.php

```diff
@@ -26,20 +27,30 @@ class SocialTaggingService {
    */
   protected $configFactory;
 
+  /**
+   * The language manager.
+   *
+   * @var \Drupal\Core\Language\LanguageManagerInterface
+   */
+  protected $languageManager;
+
   /**
    * SocialTaggingService constructor.
    *
    * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entityTypeManager
    *   Injection of the entityTypeManager.
    * @param \Drupal\Core\Config\ConfigFactoryInterface $configFactory
    *   Injection of the configFactory.
+   * @param \Drupal\Core\Language\LanguageManagerInterface $language_manager
+   *   Injection of the languageManager.
    *
    * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
    * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
    */
-  public function __construct(EntityTypeManagerInterface $entityTypeManager, ConfigFactoryInterface $configFactory) {
+  public function __construct(EntityTypeManagerInterface $entityTypeManager, ConfigFactoryInterface $configFactory, LanguageManagerInterface $language_manager) {
     $this->termStorage = $entityTypeManager->getStorage('taxonomy_term');
     $this->configFactory = $configFactory;
+    $this->languageManager = $language_manager;
```

Comment:
Reason why I call it like this, is because language_manager is the service name and this is being done throughout Drupal standards.
see
https://api.drupal.org/api/drupal/core!core.services.yml/service/language_manager/8.2.x
or for 1 of the examples under it
https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Entity%21Sql%21SqlContentEntityStorage.php/8.2.x

or for the config factory
https://api.drupal.org/api/drupal/core%21modules%21rest%21src%21LinkManager%21TypeLinkManager.php/8.2.x

so I think coding standards wise, mine is actually the way it should be.

Reply:
So we should refactor the other two :octocat:

Reply:
You are correct! Good one!